File: Lemur/src/frc/team4330/robot/commands/Climb.java
package frc.team4330.robot.commands;

import edu.wpi.first.wpilibj.command.Command;
import frc.team4330.robot.Robot;
import frc.team4330.robot.RobotMap;

public class Climb extends Command {

	
	
	@Override
	protected void initialize() {
		requires(Robot.tarzan);
	}
	
	@Override
	protected void execute() {
		Robot.tarzan.set(RobotMap.CLIMB_SPEED);
	}
	
	@Override
	protected boolean isFinished() {
		return false;
	}
	
	@Override
	protected void end() {
		Robot.tarzan.stop();
	}

}

File: Lemur/src/main/java/frc/team4330/robot/commands/Climb.kt
package frc.team4330.robot.commands

import edu.wpi.first.wpilibj.command.Command
import frc.team4330.robot.Robot
import frc.team4330.robot.RobotMap

class Climb : Command() {
    init {
        requires(Robot.tarzan)
    }

    override fun initialize() {
        Robot.tarzan.set(RobotMap.CLIMB_SPEED)
    }

    override fun isFinished(): Boolean {
        return false
    }

    override fun end() {
        Robot.tarzan.stop()
    }

}
